docker/sawtooth-dev-rust

```diff
@@ -41,24 +41,28 @@ RUN echo "deb http://repo.sawtooth.me/ubuntu/ci xenial universe" >> /etc/apt/sou
     openssl \
     pkg-config \
     python3-grpcio-tools=1.1.3-1 \
+    unzip \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
+RUN curl -OL https://github.com/google/protobuf/releases/download/v3.5.1/protoc-3.5.1-linux-x86_64.zip \
+ && unzip protoc-3.5.1-linux-x86_64.zip -d protoc3
```

Comment:
Could probably clean up the zip file once you're done with it, for image size reasons.

Reply:
Changed this so it removes the zip file. Also added -sS to curl.